android/app/src/main/java/poke/rogue/helper/presentation/dex/detail/PokemonDetailActivity.kt

```diff
@@ -261,3 +337,49 @@ class PokemonDetailActivity :
             }
     }
 }
+
+class ChipTransitionListener(
+    private val startId: Int,
+    private val endId: Int,
```

Comment:
```suggestion
class ChipTransitionListener(
    @IdRes private val startId: Int,
    @IdRes private val endId: Int,
```

@IdRes 어노테이션 추가하는거 어떠신가요?